src/components/project/controls/ProjectMonthModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import {connect} from 'react-redux';
+import {Container, Row, Form} from 'react-bootstrap';
+import moment, {Moment} from 'moment';
+import {t} from '../../utils';
+import {Modal, BaseModalProps, DatePicker} from '../../controls';
+import {ProjectMonthModel} from '../models';
+import {saveProjectMonth} from '../../../actions';
+
+type ProjectMonthModalProps = BaseModalProps & {
+  saveProjectMonth: (projectMonth: ProjectMonthModel) => void;
+};
+
+const _ProjectMonthModal = (props: ProjectMonthModalProps) => {
+  const [projectMonth, setProjectMonth] = useState<ProjectMonthModel>({
+    // ? We use the UTC method to prevent moment subtracting 1 hour, this can cause some annoying problems
```

Comment:
Delete this comment.

Also this would not have been a good comment because it is too vague.
- Why 1 hour? Why not 2 hours?
- What annoying problems could occur?